sdk/Managed/src/Microsoft.WindowsAzure.MobileServices/Table/Sync/MobileServiceSyncContext.cs

```diff
@@ -303,6 +303,28 @@ public Task CancelAndUpdateItemAsync(MobileServiceTableOperationError error, JOb
             });
         }
 
+        public Task UpdateOperationAsync(MobileServiceTableOperationError error, JObject item)
+        {
+            string itemId = error.Item.Value<string>(MobileServiceSystemColumns.Id);
+            return this.ExecuteOperationSafeAsync(itemId, error.TableName, async () =>
```

Comment:
How does this play out being called from the delegate?  Can we add a test case or two for those flows?